CONTRIBUTORS.md

```diff
@@ -0,0 +1,77 @@
+<html>

```

Comment:
@prabhakar267 
Dear sir, 
I did refer to the link provided by you and was unable to find a mandatory specification that was not covered by the pull request. It will be really helpful of you if you can point out the specification number that was not fulfilled.